1.10/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -20,6 +20,13 @@ In addition to the general [Docker requirements and recommendations for DC/OS][1
   mkfs -t xfs -n ftype=1 /dev/sdc1
   ```
 
+* If you are using Docker 1.12 or Docker 1.13, you must add `MountFlags=private` in `/etc/systemd/system/docker.service.d/execstart.conf` and then restart the Docker daemon with this command:
```

Comment:
so what should the doc say? 

If you are using Docker 1.12 or Docker 1.13, and are using either of the following features, you may encounter EBUSY errors:
- volumes exposed by plugins (containerized volume plugins)
- volumes exposed by local drivers and mounting a host directory

For more information, see https://github.com/moby/moby/issues/25718 and https://github.com/moby/moby/issues/22260.